### PR TITLE
CSS change to required text classes

### LINKED ISSUE #404: Content Creator: Visual affordance implementation
Allow users to be able to identify the fields that are required to save and those that are required to published.

```gherkin
Feature: Visual affordance for required fields
  As a Content Creator,
  I want to be able to identify which fields are required
  So that I know what fields are required when creating and editing content.

  Scenario: Required for saving
    Given I will need to gather content to enter into the CMS
      And that certain content is required for the content to be eligible to save
     When I look to create or edit content
     Then I will see a visual affordance for required fields
       And identify which fields I need to complete to save

  Scenario: Required for publishing
    Given I will need to gather content to enter on the CMS
      And that certain content is required for the content to be eligible to publish
     When I look to create or edit content
     Then I will see a visual affordance for required fields
       And identify which fields I need to complete to publish
```

---
**Visual affordance for required fields:**
- [x] Required for saving
- [x] Required for publishing

---

Reference #336

### PR DESCRIPTION
**Pull request by ineedsubstance**
This is the CSS change to adding the required text to required fields in Punchcard. I also updated the `styles` page to show this update as well.

This PR corresponds to the content-types work I did [here](https://github.com/punchcard-cms/content-types/pull/111).

---
Resolves #404 

`DCO 1.1 Signed-off-by: Nick Tilden <nick@ineedsubstance.com>`

### CODE REVIEW

File: views/styles.html [hunk 1]
```diff
@@ -192,22 +192,22 @@ <h1 class="base--h2">Form elements</h1>
...
-          <div class="form--field">
-            <label for="input__webaddress">Web Address</label>
+          <div class="form--field required--publish">
+            <label for="input__webaddress">Web Address <mark class="mark-publish">required to publish</mark></label>
             <input id="input__webaddress" type="url" placeholder="http://yoursite.com">
             <p class="form--description">This is a description of the input above</p>
           </div>
-          <div class="form--field">
-            <label for="input__emailaddress">Email Address</label>
+          <div class="form--field required--publish">
```
Comment (Snugug) [inline views/styles.html]: Why does the form field get `required--*` class? There's no reason for that. Please remove them

File: views/styles.html [hunk 2]
```diff
@@ -192,22 +192,22 @@ <h1 class="base--h2">Form elements</h1>
...
+            <label for="input__emailaddress">Email Address <mark class="mark-publish">required to publish</mark></label>
             <input id="input__emailaddress" type="email" placeholder="name@email.com">
           </div>
-          <div class="form--field">
-            <label for="input__phone">Phone Number</label>
+          <div class="form--field required--publish">
+            <label for="input__phone">Phone Number <mark class="mark-publish">required to publish</mark></label>
```
Comment (Snugug) [inline views/styles.html]: The `mark-*` class should be the `required--publish` or `required--save`.

**Comment by ineedsubstance** [comment]
@scottnath or @Snugug - getting some weird errors on the build after I updated the branch. They are js files that I did not touch. Any suggestions on how to resolve this?

File: src/sass/partials/components/_forms.scss [hunk 3]
```diff
@@ -181,18 +181,12 @@
 .required {
 
   &--save {
-
-    .mark-save {
-      background-color: setting-get('primary support color');
-      font-weight: setting-get('normal font weight');
-    }
+    background-color: setting-get('primary support color');
```
Comment (Snugug) [inline src/sass/partials/components/_forms.scss]: Are there any checks in place to ensure that the text color is accessible on the background color?

Comment (Snugug) [inline src/sass/partials/components/_forms.scss]: Nevermind, I've dug in

**Comment by Snugug** [comment]
@ineedsubstance All looks good, some errors in our tests, but restarting them now. If it all works, I'll merge!

**Pull request merged**